.github/workflows/test-pnpm.yml

```
@@ -10,22 +10,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v4
+        # Pinned to v4 commit hash for security
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)